azerothcore/azerothcore-wotlk
# fix(Scripts/Wintergrasp): fix disabled always false

## CHANGES PROPOSED:
-  Fix enable/disable Wintergrasp


## ISSUES ADDRESSED:

- Required for: https://github.com/azerothcore/azerothcore-wotlk/pull/3314


## TESTS PERFORMED:
- None


## HOW TO TEST THE CHANGES:

- Try to enable/disable Wintergrasp and make sure it's really enabled/disabled accordingly



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
